easybuild/easyconfigs/s/SeisSol/SeisSol-201703-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'SCons'
+
+name = 'SeisSol'
+local_commit_id = 'fb97bef'
+version = '201703'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.seissol.org'
+description = """SeisSol is a software package for simulating wave propagation and dynamic rupture based on
+ the arbitrary high-order accurate derivative discontinuous Galerkin method (ADER-DG)."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/SeisSol',
+        'repo_name': 'SeisSol',
+        'commit': version,
+        'recursive': True,
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = ['d0ae12400344dc596c50064a2256aaa20ead388e0639ac2fe834d48be1d59df5']
```

Comment:
We'll need to use `checksums = [None]` here, since this is downloaded via `git_config`.

We're using a specific tag, maybe we should use a specific commit ID instead (since a tag can be moved)?